python/src/opendp/_lib.py

```diff
@@ -209,7 +209,7 @@ def versioned(function):
 
         docs_ref = get_docs_ref(version)
         function.__doc__ = function.__doc__.replace(
-            "https://docs.opendp.org/en/latest/",
+            "https://docs.opendp.org/en/nightly/",
```

Comment:
This is as actually as intended, the codegen in the `opendp_tooling` crate creates the link using `latest`. (We're going to clean this up.)